packages/driver/src/cross-origin/communicator.ts

```diff
@@ -137,9 +136,30 @@ export class PrimaryOriginCommunicator extends EventEmitter {
     })
   }
 
-  toSpecBridge (origin: string, event: string, data?: any) {
+  /**
+   * Event to be sent to sent to a specific spec bridge.
+   * @param origin - the origin of the spec bridge to send the event to.
+   * @param event - the name of the event to be sent.
+   * @param data - any meta data to be sent with the event.
+   * @param responseEvent - the even to be responded with when sending back a result.
+   */
+  toSpecBridge (origin: string, event: string, data?: any, responseEvent?: string) {
     debug('=> to spec bridge', origin, event, data)
+    const source = this.crossOriginDriverWindows[origin]
+
+    if (source) {
+      this.toSource(source, event, data, responseEvent)
+    }
+  }
 
+  /**
+   * Event to be sent to sent to a specific source.
+   * @param origin - the origin of the spec bridge to send the event to.
+   * @param event - the name of the event to be sent.
+   * @param data - any meta data to be sent with the event.
+   * @param responseEvent - the even to be responded with when sending back a result.
+   */
+  toSource (source: Window, event: string, data?: any, responseEvent?: string) {
```

Comment:
Added toSource to shortcut when responding back to a post message that expects a response. This also allows us to work for both spec bridge messages and AUT messages.